benchmark/python/sparse_end2end.py

```diff
@@ -27,6 +27,8 @@
                          'otherwise, use gpu(0),...,gpu(num_gpu-1)')
 parser.add_argument('--output-dim', type=int, default=4,
                     help='number of columns of the forward output')
+parser.add_argument('--dummy-metric', type=int, default=0,
```

Comment:
Could you use `python tools/license_header.py add ` to add the missing license headers?